### ISSUE TITLE
Save file dialog's default size is too large for second monitor

### ISSUE THREAD

**Issue by dougabugg** [elementary/files#1305]
## Prerequisites
- [ ] I have searched open and closed issues for duplicates.
I searched for "save file dialog too large", no results for any open issues.

## Describe the bug
I tried to save a file, a save file dialog was opened, but the bottom of window is cut off by the screen, and I can't choose the file name or see the labels on the buttons.

I didn't see a "close" button on the window, not sure if it was cut off as well, but I had difficulty closing the save file dialog window as well.

If applicable, add screenshots or screen recordings to help explain your problem.

[Screenshot from 2020-04-28 15-00-48](https://user-images.githubusercontent.com/18294318/80532606-07453d80-8962-11ea-851b-5fdcdce32b31.png)

### To Reproduce

Have a second monitor which has a smaller vertical resolution than the primary monitor, and invoking the save file dialog from an app that is on the second monitor.

### Expected behavior

I expected to be able to see the bottom of the window and choose the name of the file and click the save button.

## Platform Information

- [ ] I'm using the latest version from git that I've manually compiled
- [ ] I'm using the latest released stable version

`uname -a:
Linux my-computer 5.3.0-46-generic #38~18.04.1-Ubuntu SMP Tue Mar 31 04:17:56 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by cassidyjames** [comment]
fwiw it looks like the second display is ~1366×768 from my pixel-measuring in Glimpse :smile:

**Comment by jeremypw** [comment]
Looks like Files has nothing to do with the initial size and position of the dialog - it just modifies an existing Gtk.FileChooserDialog window by swapping out the pathbar for a Files-style one and modifying the behaviour a little. Which app was calling the save dialog? It should pass a parent window to the dialog.  It is also possiblly related to this other problem handling a second smaller monitor: https://github.com/elementary/gala/issues/764

**Comment by cassidyjames** [comment]
@jeremypw I don't think that linked issue is related at all, as it just has to do with apps that are pinned to all workspaces (which according to Mutter includes all windows on secondary displays).
